queue_job/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+{'name': 'Job Queue',
+ 'version': '10.0.1.0.0',
+ 'author': 'Camptocamp,Acsone,Odoo Community Association (OCA)',
+ 'website': 'http://odoo-connector.com',
+ 'license': 'AGPL-3',
+ 'category': 'Generic Modules',
+ 'depends': ['mail'
+             ],
+ 'external_dependencies': {'python': ['requests'
+                                      ],
+                           },
+ 'data': ['security/security.xml',
+          'security/ir.model.access.csv',
+          'views/queue_job_views.xml',
+          'data/queue_data.xml',
+          ],
+ 'installable': True,
+ 'application': True,
```

Comment:
@blaggacao if the user has not the accesses rights for the queue, he won't see the top-level menu, so for a technical end-user, it can make sense to have a top-level menu (jobs is a menu you open often as you regularly want to check them, it should not be hidden in an obscure place.